packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -1,67 +1,61 @@
-/* eslint sort-keys: "error" */
-
-module.exports = {
-  "proposal-async-generator-functions": "Asynchronous Iterators",
-  "proposal-json-strings": "JSON superset",
-  "proposal-nullish-coalescing-operator": "nullish coalescing operator (??)",
-  "proposal-object-rest-spread": "object rest/spread properties",
-  "proposal-optional-catch-binding": "optional catch binding",
-  "proposal-optional-chaining": "optional chaining operator (?.)",
-  "proposal-unicode-property-regex": "RegExp Unicode Property Escapes",
+const es5 = {
```

Comment:
Would be good to add a comment here if the key order will impact the plugin execution order of preset-env.